fdbserver/FDBExecHelper.actor.cpp

```diff
@@ -83,7 +83,9 @@ ACTOR Future<int> spawnProcess(std::string binPath,
 
 static auto fork_child(const std::string& path, std::vector<char*>& paramList) {
 	int pipefd[2];
-	pipe(pipefd);
+	if (pipe(pipefd) != 0) {
+		_exit(EXIT_FAILURE);
```

Comment:
Can we change this to return an error pid:

```suggestion
		return std::make_pair(-1, Optional<int>{});
```